pandas/core/frame.py

```diff
@@ -1847,6 +1847,10 @@ def to_dict(self, orient: str = "dict", into=dict):
             instance of the mapping type you want.  If you want a
             collections.defaultdict, you must pass it initialized.
 
+        index : bool, default True
+            When set to False, method returns a dict without an index key
+            (and index_names if using orient='tight').
```

Comment:
Needs a versionadded marker

Reply:
@phofl the versionadded should be right on the line above the index right?